commit ba118b3e907673ebc68dfe7dfcf45dad829404ea (refs/changes/08/4708/1)
Author: Josh Wilsdon <josh@wilsdon.ca>
Date:   2018-08-16T17:17:11-07:00 (1 year, 2 months ago)
    
    TRITON-703 NAPI should use latest node-triton-metrics and expose nodejs metrics

diff --git a/lib/napi.js b/lib/napi.js
index 6d1147b..11bd9ff 100644
--- a/lib/napi.js
+++ b/lib/napi.js
@@ -108,6 +108,7 @@ function NAPI(opts) {
     });
 
     this.metricsManager.createRestifyMetrics();
+    this.metricsManager.createNodejsMetrics();
     this.metricsManager.listen(function metricsServerStarted() {});
 
     function populateReq(req, res, next) {
diff --git a/package.json b/package.json
index 8cb37b7..2d66ec2 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "napi",
   "description": "SmartDataCenter Networking API",
-  "version": "1.3.0",
+  "version": "1.3.1",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -24,7 +24,7 @@
     "sdc-clients": "11.1.0",
     "tape": "4.5.1",
     "trace-event": "1.3.0",
-    "triton-metrics": "0.1.1",
+    "triton-metrics": "0.4.0",
     "vasync": "1.6.4",
     "verror": "1.9.0"
   },
